### PR TITLE
Edited the Getting Started parts of the documentation for grammar, clarity, and style

### PR DESCRIPTION
**Pull request by HopefulFrog**
One of the main changes I made was removing references to "we" and "let's", so as not to directly reference the author of the documentation, to maintain formality.

I also made other various edits to grammar, sentence structure, and so on.

If these changes are acceptable, I can do the same for the rest of the documentation.

### CODE REVIEW

File: Documentation/articles/getting_started/2_creating_a_new_project_vs.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 ![New Solution](~/images/getting_started/vswin-mg-new-1.png)
 
-Now you should see a "New Project" dialog pop up, from here select **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**.
+You should see the "Create a new project" dialog pop up. From here, select the **Templates > Visual C# > MonoGame** category, and then select **MonoGame Cross Platform Desktop Project**.
 
 ![New Template](~/images/getting_started/vswin-mg-new-2.png)
 
-Next type in the name that you wish to give your project, for this tutorial let's just use **Pong** (*note* you should not use spaces when naming the project). After you've entered the name, click on the **...** button next to the location text field, and select where you wish to save your project. Finally click **OK** to create a new project.
+Next, type in a name for your project. For this tutorial, **Pong** will be used (note: project names cannot contain spaces). After you've entered the name, click on the **...** button next to the Location text field and choose a directory to contain your project. Finally, click **OK** to create the project.
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/2_creating_a_new_project_vs.md:13]: > choose a directory to contain your project.

That doesn't sound right.

File: Documentation/articles/getting_started/2_creating_a_new_project_vs.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
 ![Project Name](~/images/getting_started/vswin-mg-new-3.png)
 
-If everything went correctly, you should see an **Pong** project open up like in the picture below. To run your game simply press the big **Play Button** in the toolbar or press **F5**.
+If everything went correctly, you should see a project named **Pong** (or whatever name you decided on) open up like in the picture below. To run your game, simply press the big **Play Button** in the toolbar, or press **F5**.
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/2_creating_a_new_project_vs.md:17]: > (or whatever name you decided on)

Feels redundant as we specified that we will use Pong as our project name for the tutorial.

File: Documentation/articles/getting_started/2_creating_a_new_project_vsm.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ Start Visual Studio for Mac and select **New** on the right side.
 
 ![New Solution](~/images/getting_started/vsmac-mg-new-1.png)
 
-Now you should see a "New Project" dialog pop up. From here select **MonoGame > App** category, then select **MonoGame Cross Platform Desktop Project** and click **Next**.
+Now you should see a "New Project" dialog pop up. From here, select the **MonoGame > App** category, then select **MonoGame Cross Platform Desktop Project** and click **Next**.
 
 ![New Template](~/images/getting_started/vsmac-mg-new-2.png)
 
-On the following dialog, type in the name that you wish to give your project, for this tutorial let's just use **Pong** (*note* you should not use spaces when naming the project). After you've entered the name, click on the **Browse** button next to the location text field, and select where you wish to save your project. Finally, click **Create** to create a new project.
+On the following dialog, type in a name for your project. For this tutorial, **Pong** will be used (note: project names cannot contain spaces). After you've entered the name, click on the **Browse** button next to the Location text field and choose a directory to contain your project. Finally, click **Create** to create a new project.
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/2_creating_a_new_project_vsm.md:13]: Same comment from above.

File: Documentation/articles/getting_started/2_creating_a_new_project_vsm.md [hunk 4]
```diff
@@ -6,15 +6,15 @@ Start Visual Studio for Mac and select **New** on the right side.
...
 
 ![Project Name](~/images/getting_started/vsmac-mg-new-3.png)
 
-If everything went correctly, you should see a **Pong** project open up like in the picture below. To run your game simply press the big **Play Button** in the upper left corner or press **F5**.
+If everything went correctly, you should see a project named **Pong** (or whatever name you decided on) open up like in the picture below. To run your game, simply press the big **Play Button** in the upper left corner, or press **F5**.
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/2_creating_a_new_project_vsm.md:17]: Same comment from above.

File: Documentation/articles/getting_started/4_adding_content.md [hunk 5]
```diff
@@ -2,29 +2,29 @@
 
 This tutorial will go over adding content to your game.
 
-> For help on creating a project please look at the [Creating a New Project](0_getting_started.md) of the Getting Started guide.
+> For help with creating a project, please look at the [Creating a New Project](0_getting_started.md) section of the Getting Started guide.
 
 ## MonoGame Content Builder Tool (MGCB Editor)
 
-If you have not already, you will need to install the MGCB-Editor for managing your content, for details on getting access to the tool, see the [installation instructions here](~/articles/tools/mgcb_editor.md).
+If you haven't already, you'll need to install the MGCB-Editor for managing your content. For details on getting access to this tool, see the [installation instructions here](~/articles/tools/mgcb_editor.md).
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/4_adding_content.md:9]: MGCB-Editor -> MGCB Editor

File: Documentation/articles/getting_started/4_adding_content.md [hunk 6]
```diff
@@ -2,29 +2,29 @@
...
+You should now see the MGCB Editor window open up. If a text file opens instead, then right-click on **Content.mgcb** and select **Open With**, then select **mgcb-editor-wpf** in the list, click **Set as Default** and then click **OK**, then try again.
 
-> If you do not see the **mgcb-editor-wpf** option when you right-click and select **Open With**, then please review the [Tools documentation](~/articles/tools/tools.md) for installing the MGCB Editor tool for your operating system.
+> If you do not see the **mgcb-editor-wpf** option when you right-click and select **Open With**, then please review the [Tools documentation](~/articles/tools/tools.md) for installing the MGCB Editor tool for your operating system. If you do see the option, but setting it as default still results in a text file being opened when **Content.mgcb** is clicked, try right-clicking it instead and selecting **Open Containing Folder**. This will open File Explorer, and from there you should be able to double click the MGCB file to open up the editor.
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/4_adding_content.md:27]: You don't need to open file explorer, you can use open with, select mgcb-editor-wpf, and than click open. There is a bug in VS that doesn't want to properly set the default editor.

File: Documentation/articles/getting_started/4_adding_content.md [hunk 7]
```diff
@@ -61,7 +61,7 @@ public class Game1 : Game
     private SpriteBatch _spriteBatch;
 ```
 
-Next find the LoadContent method and use it to retrieve the "ball" sprite from the Content project into the **ballTexture** private variable using the **Content.Load()** method and specifying the type of data we are requesting, in this case a Texture2D image:
+Next, find the LoadContent method. Here, retrieve the "ball" sprite from the Content project and store it in the **ballTexture** private variable using the **Content.Load()** method, with the type of content you're loading specified in the angle brackets—in this case, a Texture2D image:
```
Comment (harry-cpp) [inline Documentation/articles/getting_started/4_adding_content.md:64]: Very awkward wording.

**Review by harry-cpp — CHANGES_REQUESTED** [review]
Great work @HopefulFrog ! I left a couple of notes, once those are fixed I'll merge this asap.

**Comment by HopefulFrog** [comment]
@harry-cpp I've made changes in an effort to address the problems you brought up.

**Comment by harry-cpp** [comment]
Thanks @HopefulFrog , merging!

**Pull request merged**